modules/orchestrator/proc-install-orchestrator-plugin.adoc

```diff
@@ -0,0 +1,75 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-install-orchestrator-plugin_{context}"]
+= Installing the Orchestrator plugin using Operator
+
+You can enable the Orchestrator plugin in {product-very-short} by configuring dynamic plugins in your {product-custom-resource-type} custom resource (CR).
+
+.Prerequisites
+* You have installed {product-very-short} on {ocp-short}.
+* You have access to edit or create ConfigMaps in the namespace where the {product-custom-resource-type} CR is deployed.
+
+.Procedure
+. Identify the Orchestrator plugin dynamic plugins that are included by default in the default `dynamic-plugins.yaml` file of the `install-dynamic-plugins` container:
```

Comment:
File name if 'dynamic-plugins.default.yaml' 

Side comment: I believe "Identify the Orchestrator plugin dynamic plugins ... in the init container" is not trivial procedure, better if we have some place where user were able to read the list of release plugins (with versions)